### ISSUE TITLE
Bug: "ValueError: Length of values does not match length of index" in get_trip_summaries

### ISSUE THREAD

**Issue by rmitsch** [univie-datamining-team3/assignment2#10]
Only if chosen token is not KEY_LUKAS. 
Reason: Mismatch between number of start/end times and number of trips, since some trips apparently don't contain annotation data (e. g. no time or transport mode) and hence are useless.

Proposed solution: Method `_remove_dataframes_without_annotation(dataframes)` dropping all trips w/o annotation data.

**Comment by rmitsch** [comment]
Implemented as `Preprocessing._remove_dataframes_without_annotatation(dataframes)` (in my branch).
@Lumik7 @MoBran Closing as resolved if you agree with the proposed solution.

**Comment by Lumik7** [comment]
I already fixed this bug and pushed a solution to the master branch. As I wrote my solution prior to talking to you about invalid recordings, I did not drop the data and initialised the annotations with empty values. This solution can be seen in the [get_trip_summaries](https://github.com/univie-datamining-team3/assignment2/blob/master/src/data/data_utils.py) method.

Anyway, now that we know that this are useless recordings they can be dropped.
